src/tabstyle.cpp

```diff
@@ -63,7 +63,7 @@ void TabStyle::drawControl(ControlElement element,
         // Draw the final text
         drawItemText(painter,
                      rect,
-                     Qt::AlignVCenter,
+                     Qt::AlignCenter,
```

Comment:
Sorry for being late. Is this intentional? Now texts are aligned horizontally center, which looks strange to me, especially when there are icons.
![default](https://user-images.githubusercontent.com/1937689/35875894-a33edd04-0bab-11e8-9915-61d15f7608d4.png)

I like the original left-aligned texts more.

Reply:
Yes, it's intentional, because there's an extra space for the text of inactive tabs. Horizontal symmetry is more elegant than than the lack of it; why does it look strange to you? Let me guess: because of Fusion and Breeze ;)

Reply:
I guess I want to get consistency between my terminal and my browser. Here's how Firefox draws a tab:
![default](https://user-images.githubusercontent.com/1937689/35876407-23d5eeb6-0bad-11e8-8e68-83716038bf8e.png)
Whether it's more elegant or not, I'm accustomed to it.

Reply:
Terminal and browser?! Qt app and Firefox?

Change it to whatever seems better to you if others agree -- I won't disagree because this app isn't mine.

Reply:
Just to show that such things aren't absolute, this is my browser:

![browser](https://user-images.githubusercontent.com/981076/35876828-c3e9eaee-0b88-11e8-9bd1-195671ee2d46.png)

Reply:
Oh, you just reminds me that it's possible to load custom QSS #308. I'll check that out.

Reply:
Wait a minute! Konsole has centered tab texts too. This screenshot is taken with Breeze but it's the same with Fusion:

![konsole](https://user-images.githubusercontent.com/981076/35877334-42cae664-0b8a-11e8-9b72-859d9982a0ca.png)

It seems the centered tab text is a terminal tradition I respected unaware ;)